### ISSUE TITLE
Unable to compile on iOS

### ISSUE THREAD

**Issue by piscis** [bratan/flutter_device_locale#3]
Hi I just noticed that, this plugin does not compile on flutter swift projects.
```bash

Running pod install...                                              1.5s
Running Xcode build...                                                  
                                                   
Xcode build done.                                            2.7s
Failed to build iOS app
Error output from Xcode build:
↳
    ** BUILD FAILED **


Xcode's output:
↳
    /Users/x/y/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_device_locale-0.3.4+1/ios/Classes/FlutterDeviceLocalePlugin.m:2:9: fatal error:
    'flutter_device_locale/flutter_device_locale-Swift.h' file not found
    #import <flutter_device_locale/flutter_device_locale-Swift.h>
            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    1 error generated.
    note: Using new build system
    note: Planning build
    note: Constructing build description

```

**Comment by bratan** [comment]
I can't reproduce your issue, here everything compiles fine on both Xcode and IntelliJ

```
Launching lib/main.dart on iPhone 11 Pro Max in debug mode...
Running Xcode build...
Xcode build done.                                           14.9s
Syncing files to device iPhone 11 Pro Max...
```

Did you try to run the example project?

https://github.com/bratan/flutter_device_locale/tree/master/example

**Comment by piscis** [comment]
Hi I tried the example and it works so I guess it's something to a xcode workspace related setting on my side.

**Comment by marco2250** [comment]
> Hi I tried the example and it works so I guess it's something to a xcode workspace related setting on my side.

@piscis were you able to solve the problem?

**Comment by marco2250** [comment]
> I can't reproduce your issue, here everything compiles fine on both Xcode and IntelliJ
> 
> ```
> Launching lib/main.dart on iPhone 11 Pro Max in debug mode...
> Running Xcode build...
> Xcode build done.                                           14.9s
> Syncing files to device iPhone 11 Pro Max...
> ```
> 
> Did you try to run the example project?
> 
> https://github.com/bratan/flutter_device_locale/tree/master/example

Dear @bratan , may you help me?

I understand that you couldn't reproduce the problem, but unfortunately I am having it...

So, I would like to ask why the file "FlutterDeviceLocalePlugin.m" imports a file "flutter_device_locale-Swift.h" but we can't find it in the project?

I guess the answer may help me find what I'm doing wrong.

Thank you in advance.

**Comment by jingyu6lu** [comment]
> Hi I tried the example and it works so I guess it's something to a xcode workspace related setting on my side.

Hello I want to know did you solved the issue? thanks

**Comment by piscis** [comment]
> > Hi I tried the example and it works so I guess it's something to a xcode workspace related setting on my side.
> 
> Hello I want to know did you solved the issue? thanks

No I did not eventually in my case I ended up with a different package "device_info"

**Comment by Keetz** [comment]
I am getting this error too.
It might have been fixed for new projects, but how do I make this work on an older project?

I am using latest version of the plugin, but it is on a older project which I first tried building through xcode yesterday. How do I update my project or this plugin to make it work? It is not an option to create a new project.

***EDIT***
I got it working by adding **use_frameworks!** to the **target 'Runner' do**
